src/types/global.d.ts

```diff
@@ -1,7 +1,7 @@
 declare global {
   interface ImportMeta {
     env: {
-      NODE_ENV: 'development' | 'production' | 'test'
```

Comment:
With `/// <reference types="vite/client" />` you wouldn't need this line, as `PROD` is a "built-in" constant in vite.

Reply:
+1 moving that into https://github.com/oasisprotocol/explorer/issues/2040